code/src/app.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background-color: black;
+  color: white;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.album-cards {
+  position: relative;
+}
+.image-container {
+  position: relative;
+}
+
+.image-container:hover {
+  opacity: 0.4;
+}
+
+.album-cards:hover .icons {
+  display: flex;
+}
+
+.icons {
+  position: absolute;
+  z-index: 5;
+  width: 50%;
+  fill: white;
+  display: flex;
+  justify-content: space-between;
+  top: 40%;
+  left: 30%;
+  display: none;
+}
+
+.heart,
+.dot {
+  width: 30px;
+}
+.play {
+  width: 60px;
+  transition: 0.4s ease;
+}
+.play:hover {
+  transform: scale(1.3);
+}
+
+.album-container {
+  max-width: 1200px;
+  grid-template-columns: repeat(1, 1fr);
+  display: grid;
+  grid-gap: 1rem;
+  position: relative;
+  background-size: cover;
+  min-height: 50vh;
+  padding: 20px;
+  margin: 0 auto;
+}
+
+@media (min-width: 668px) {
+  .album-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 1024px) {
+  .album-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
```

Comment:
The `album-card`s should probably be named `album-card` - because they all contain one card each, right? And they don't fit in the `album-container` in small desktop size so you get a horizontal scroll:
<img width="1064" alt="Skärmavbild 2020-10-26 kl  11 28 46" src="https://user-images.githubusercontent.com/51529201/97161658-6bfd1b80-177e-11eb-9af9-c5fe3cf06881.png">